packages/better-auth/src/context/create-context.ts

```diff
@@ -104,6 +104,12 @@ export async function createAuthContext(
 	const logger = createLogger(options.logger);
 	const baseURL = getBaseURL(options.baseURL, options.basePath);
 
+	let displayError = false;
```

Comment:
I think this is unnecessary since it will only be called once per `betterAuth` call. And it won't work if there are multiple initial unless you put it in `globalThis`